### PR TITLE
Add donut.me chat apps to showcase

### PR DESCRIPTION
**Pull request by dbrugne**
Adding donut.me chat iOS/Android apps to showcase.

### CODE REVIEW

**Comment by facebook-github-bot-3** [comment]
By analyzing the blame information on this pull request, we identified @brentvatne and @manonthemat to be potential reviewers.

**Comment by facebook-github-bot-3** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

**Comment by facebook-github-bot-8** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

File: website/src/react-native/showcase.js [hunk 1]
```diff
@@ -363,6 +363,13 @@ var apps = [
     author: 'Genki Takiuchi (s21g Inc.)',
   },
   {
+    name: 'DONUT chatrooms for communities',
+    icon: 'http://a2.mzstatic.com/eu/r30/Purple49/v4/d4/2d/e5/d42de510-6802-2694-1b60-ca80ffa1e2cb/icon175x175.png',
+    linkAppStore: 'https://itunes.apple.com/fr/app/donut-chat/id1067579321',
+    linkPlayStore: 'https://play.google.com/store/apps/details?id=me.donut.mobile',
+    author: 'Damien Brugne'
```
Comment (dsibiski) [inline website/src/react-native/showcase.js]: Can you please add a trailing comma here? As per the styleguide: https://github.com/facebook/react-native/blob/master/CONTRIBUTING.md#general

**Comment by facebook-github-bot-7** [comment]
@dbrugne updated the pull request.

**Comment by dbrugne** [comment]
Trailing comma added, sorry.

**Comment by dsibiski** [comment]
@facebook-github-bot shipit

**Comment by dsibiski** [comment]
Thanks @dbrugne 👍

**Comment by facebook-github-bot-2** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/1141909245861278/int_phab) to review.

**Pull request closed without merging**